src/Native/Runtime/RuntimeInstance.cpp

```diff
@@ -822,7 +822,13 @@ enum RuntimeHelperKind
 // Create indirections for all helpers used below
 
 #define DECLARE_INDIRECTION(HELPER_NAME) \
-    EXTERN_C void * HELPER_NAME; \
+    EXTERN_C void HELPER_NAME(); \
```

Comment:
Could you do this like e.g. the `COOP_PINVOKE_HELPER` macro (argument list is another parameter to the macro)? Let's try to avoid introducing 3 macros for this.

Reply:
Yes, that's better.  Made that change